wordpress-mobile/WordPress-iOS
# Quick Start V2: Congratulation screen

Refs. #10860 

This PR adds the success screen to the tour checklist.

![qs-01](https://user-images.githubusercontent.com/912252/52059153-f4766a00-2560-11e9-841b-ae87d2e272a3.gif)

![qs-02](https://user-images.githubusercontent.com/912252/52059175-fd673b80-2560-11e9-8aec-aa8ce3555242.gif)

@ScoutHarris I used the no result screen as tableview background view. It works perfectly, but I added two new public methods to expose the 2 private methods I needed. This because some method like the `viewWillAppear` and `viewWillTransition` are not triggered in this case.

**To test:**
- Put return true in shouldShowQuickStartCheacklist()
- In `QuickStartTourGuide`, _customizeListTours_, remove `QuickStartSiteIconTour()` and `QuickStartNewPageTour()` in order to complete the checklist
- As soon the list is completed you should see the congratulation screen
- Expanding the completed section, should hide/show the congratulation screen
